apps/hugging-face/src/services/huggingfaceImage.ts

```diff
@@ -20,6 +20,9 @@ export async function generateImage(prompt: string, parameters: AppInstallationP
 
     return image;
   } catch (error) {
-    throw new Error('Failed to generate image with image model');
+    const errorMessage = (error as any)?.message?.includes('exceeded')
+      ? 'You have exceeded your monthly included Huggingface credits.'
```

Comment:
Should "Huggingface" be written as "Hugging Face" instead?  That is how they write it on their website and I think it's always a good idea to match their branding: https://huggingface.co/brand